### ISSUE TITLE
DSMR-logger

### ISSUE THREAD

**Issue by mrWheel** [TychoSchenkeveld/com.p1.dsmr#2]
Hallo Tycho,

Ik heb geen enkele ervaring met Homey maar denk dat een aantal bouwers hier wel interesse in hebben.
Mag ik op mijn [website](https://willem.aandewiel.nl/index.php/2019/04/09/dsmr-logger-v4-slimme-meter-uitlezer/) een link naar deze repo zetten?
Kun je wat screenshots maken zodat het voor de lezer wat toegankelijker wordt?

**Comment by TychoSchenkeveld** [comment]
Bedankt voor de reactie. Sorry voor de late reply, ik dacht dat ik notificaties zou krijgen maar dat gebeurde niet. Bedankt voor het ontwerpen van het board!

Inmiddels heeft de vriend voor wie ik het maakte geen Homey meer, dus ik kan het niet meer verder ontwikkelen. Wellicht kan iemand anders het overnemen.

**Comment by BardoelT** [comment]
Hoi @TychoSchenkeveld , sinds kort heb ik een Homey en een DSMR-logger van @mrWheel. Ik heb een eigen app gemaakt met ondersteuning voor de nieuwe REST API, maar die wordt niet in de Homey app store toegelaten omdat jouw app er al in staat. Bij Athom willen ze geen duplicate apps, wat ik me wel voor kan stellen. Ze geven de suggestie om verder te ontwikkelen op de bestaande app, of de ownership over te dragen zodat ik er verder mee kan. 

Sta jij ervoor open om de ownership over te dragen?

**Comment by TychoSchenkeveld** [comment]
Hoi Thijs! Ja prima! Ik doe er niets meer mee, ik heb het voor een vriend gemaakt maar die heeft er geen meer en ik heb zelf nooit een Homey gehad. Zelf heb ik Home Assistant.

Ik kan deze ook intrekken als je wil.

**Comment by TychoSchenkeveld** [comment]
Als het goed is ben je nu owner ervan!

**Comment by BardoelT** [comment]
Ja, dat is gelukt! Nu zit alleen de Homey app nog op jouw developers account, is het mogelijk om daar iets te delen/over te dragen?

**Comment by TychoSchenkeveld** [comment]
Ja dat kan! Ik heb alleen je developer ID nodig (lange hex string).

**Comment by BardoelT** [comment]
Mijn CloudID is 5eaba54c98dade0c7b3282fd

**Comment by TychoSchenkeveld** [comment]
Hmm when I try to transfer it says "A user with this ID doesn't exist". Are you sure you have enabled the developer functionality?

**Comment by BardoelT** [comment]
deze dan? 5ed512176e029d72ba37bb94

**Comment by TychoSchenkeveld** [comment]
Ja die ging wel goed! Zou nu gebeurd moeten zijn.

En veel succes met de app! 👍

**Comment by TychoSchenkeveld** [comment]
PS heb de berichten met de ID's verwijderd voor het geval dat..

**Comment by BardoelT** [comment]
Super bedankt! 👍
